lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -811,46 +801,57 @@ minWaitNumberForExtract(ttg::ExtractSliceOp extractOp) {
       int min2 = minOverHistories(prevVal, yieldOp, thisHistorySum);
       return std::min(std::min(min1, min2), minCommitNumber);
     }
-    // Failed to track, return 1 conservatively.
-    return 1;
+    // Failed to track, return 0 conservatively.
+    return 0;
   };
 
-  int minCommits = minOverHistories(extractOp.getOperand(0), extractOp, 0);
-  if (minCommits == 0)
-    llvm::report_fatal_error("No commits between insert and extract!");
-  return minCommits - 1;
+  if (waitOp->getNumOperands() != 1)
```

Comment:
Can this happen?

Reply:
technically yes, as the wait takes a variadic number of tokens.

Reply:
Should we have an assert? This would be unexpected, right?

Reply:
unexpected in the current flow I guess, still correct